pyta_server/routes.py

```diff
@@ -0,0 +1,32 @@
+import os
+import secrets
+import psycopg2
+from flask import request
+from pyta_server import app, db
+from pyta_server.models import Uploads
+
+@app.route('/', methods=['POST'])
+def receive():
+    if request.method == 'POST':
+        dirs = {'source':'', 'config':''}
+        rand_hex = secrets.token_hex(8)
+        for f_type in request.files.keys():
+            user_f = request.files[f_type]
+            _, f1_ext = os.path.splitext(user_f.filename)
+            fn = rand_hex + f1_ext
+            f_path = os.path.join(app.root_path, f'static\{f_type}', fn)
```

Comment:
This needs to be changed to separate out `static` and the path. Just had `'static'` be a separate argument.